taca/analysis/analysis_nanopore.py

```diff
@@ -416,6 +367,159 @@ def ont_updatedb(ont_run):
         send_mail(email_subject, email_message, email_recipients)
 
 
+def transfer_html_report(ont_run):
```

Comment:
Maybe this could be moved into one of the ONT transfer classes?

Reply:
And come to think of it, maybe ont_updatedb() is also better suited in a class?

Reply:
76c1a307c0fb1572f07a72085c3774624efc707d